aten/src/ATen/native/Convolution.cpp

```diff
@@ -3,6 +3,9 @@
 
 #include "ATen/Config.h"
 
+static const int MIOPEN_DIM_MAX = 4;
+static const bool MIOPEN_ENABLED = getenv("DISABLE_MIOPEN") == NULL;
```

Comment:
No, we're not going to do this. Please plumb through `miopen_enabled` in the same way `cudnn_enabled` is currently done, if you want to be able to disable it. (But I'm not sure why you would want to disable it.)

Reply:
It makes debugging easier to be able to switch w/o recompile. But I'll remove it in this PR.